devtools-har-extension/contentscript.js

```diff
@@ -0,0 +1,31 @@
+/* global chrome */
+(function (window, chrome) {
+  window.__sendPageTimings = function () {
+    var page = {
+      'pageTimings': {
+        'onContentLoad': window.performance.timing.domContentLoadedEventEnd - window.performance.timing.fetchStart,
+        'onLoad': window.performance.timing.loadEventEnd - window.performance.timing.fetchStart
+      },
+      'startedDateTime': new Date(window.performance.timing.fetchStart).toISOString(),
+      'title': window.location.href
+    }
+
+    chrome.runtime.sendMessage({
+      source: 'devtools-har-extension',
+      type: 'page',
+      data: page
+    })
+  }
+
+  window.addEventListener('load', function () {
+    setTimeout(function () {
+      window.__sendPageTimings()
+    }, 1)
+  })
```

Comment:
Because on the exact moment when the `load` event is fired the `window.performance.timing.loadEventEnd` field was not updated yet and still `0`.
So I delayed the `__sendPageTimings ` call in one millisecond.